app/views/admin/users/email_message.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="grid gap-6">
+  <div class="crayons-card p-6">
+    <h2>Email message</h2>
+
+    <table class="crayons-table mt-5" width="100%">
+      <tbody>
+        <tr>
+          <td>Subject</td>
+          <td><%= @email.subject %></td>
+        </tr>
+        <tr>
+          <td>To</td>
+          <td><%= @email.to %></td>
+        </tr>
+        <tr>
+          <td>Feedback Message</td>
+          <td><%= @email.feedback_message_id.present? ? @email.feedback_message_id.present : "N/A" %></td>
+        </tr>
+        <tr>
+          <td>Sent at</td>
+          <td><%= @email.sent_at&.strftime("%b %e '%y") %></td>
+        </tr>
+        <tr>
+          <td>Opened At</td>
+          <td><%= @email.opened_at&.strftime("%b %e '%y") %></td>
+        </tr>
+        <tr>
+          <td>UTM Campaign</td>
+          <td><%= @email.utm_campaign %></td>
+        </tr>
+        <tr>
+          <td>Mailer</td>
+          <td><%= @email.mailer %></td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  <div class="crayons-card p-6">
+    <h2>Email content</h2>
+    <p><em>The content is previewed below without formatting</em></p>
+    <div class="crayons-card my-5 p-5">
+      <%= @email.body_html_content.html_safe %>
```

Comment:
Codeclimate is not happy about rendering the email content as `html_safe`. I don't think we should see any problems with this because this is an admin tool and it's not open for the public to abuse.

That being said, I'm okay with reverting to present this without `html_safe` if anyone feels like that would be better